app/src/test/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailPresenterTest.kt

```diff
@@ -83,6 +83,11 @@ class ProgramEventDetailPresenterTest {
             BoundingBox.fromLngLats(0.0, 0.0, 0.0, 0.0),
             listOf()
         )
+        val mapData = ProgramEventMapData(
+            mutableListOf(),
+            mutableMapOf("key" to FeatureCollection.fromFeature(Feature.fromGeometry(null))),
+            BoundingBox.fromLngLats(0.0, 0.0, 0.0, 0.0)
```

Comment:
I think you don't need here BoundingBox.fromLngLats(0.0, 0.0, 0.0, 0.0) as I commented above

Reply:
The test had it like that before... so i just added the mapOf parameters. The presenter was using a Quartet class to pass all thouse including the bounding box, i just created a data class instead as it can be updated way easier. So I would leave this one as it is